entries/normalizer.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<entry name="normalizer" type="method" return="String">
+	<title>normalizer</title>
+	<desc>Change/transform the value of the element before validating.</desc>
+	<longdesc>
+		The user can change the value of an element before validating the element in question. The new value will be then used by the associated methods instead of the real one.
```

Comment:
other entries also dont repeat in long-desc what was already mentioned in the desc.

I think we could just mention the restrictions (string type) and the arguments beeing passed to it (the value) and the adjusted "this" context